test/parallel/test-util-inspect.js

```diff
@@ -690,6 +690,20 @@ util.inspect({ hasOwnProperty: null });
   };
 
   util.inspect(subject, { customInspectOptions: true });
+
+  // util.inspect.custom is a shared symbol which can be accessed as
+  // Symbol.for("util.inspect.custom").
+  const inspect = Symbol.for('util.inspect.custom');
```

Comment:
Maybe

```js
assert.strictEqual(util.inspect.custom, inspect);
```

also?

Reply:
I could do that, but it feels rather like `assert(true === true)`. If it fails, the other tests will fail. If it passes, the other tests are redundant.

Reply:
I'd prefer this test over the other ones then.

Reply:
@chocolateboy it is not like `assert(true === true)` because it verifies the reference equality and that is the most important part here. So yes, the other tests are redundant in that case but that should already highlight that this test is actually preferable over the others.

Reply:
@BridgeAR, @TimothyGu 

> it is not like `assert(true === true)` because it verifies the reference equality

I'm not clear how that differs from or improves on `assert(true === true)`.

> that is the most important part here

I disagree. I think this change violates a tenet of testing: [Test behavior, not implementation](https://testing.googleblog.com/2013/08/testing-on-toilet-test-behavior-not.html).

It's possible for:

```javascript
util.inspect.custom === Symbol.for("util.inspect.custom")
```

\- to be true and for these tests to fail because `inspect` accidentally or maliciously ends up checking a different symbol. At the moment, this mismatch would be caught by the `util.inspect.custom` tests preceding these tests, but that's just an unintended consequence of the way the tests are (or [aren't](http://wiki.c2.com/?UnitTestIsolation)) organized.

The equality of the references is not the goal: it's the means to an end (using the hook outside of Node.js). If JavaScript supported operator overloading or custom hash codes, they could even be different references. It's the "Does this work outside of Node.js?" that matters, and that's what these tests test.